ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/FunctionArgumentsUtil.java

```diff
@@ -86,29 +85,29 @@ public KsqlScalarFunction getFunction() {
 
   public static final class ArgumentInfo {
     final SqlArgument sqlArgument;
-    final TypeContext context;
+    final Map<String,SqlType> lambdaMapping;
```

Comment:
nit: call this and the getter (get)lambdaParamTypeMapping

Reply:
changed it to (get)lambdaSqlTypeMapping